src/loaders/tvg/tvgTvgLoadParser.cpp

```diff
@@ -502,6 +504,15 @@ static LoaderResult _parsePaint(tvgBlock base_block, Paint **paint)
 /* External Class Implementation                                        */
 /************************************************************************/
 
+bool tvgValidateTvgHeader(const char *ptr, uint32_t size)
+{
+    auto end = ptr + size;
+    if (_readTvgHeader(&ptr) && ptr < end) {
```

Comment:
keep it one line.
if (...) return true;

Reply:
Fixed